ReleaseNotes.md

```diff
@@ -2,6 +2,22 @@
 
 ## 1.44.0
 
+### layerselector2
+
+Fixed filter buttons to show each tabs. Also fixed undefined error for ShowFilteredLayerListRequest.
+
+Changed ``stats`` filter name to ``vectors`` (because it's filter vector layers).
+```javascript
+var sb = Oskari.getSandbox();
+var reqBuilder = sb.getRequestBuilder('ShowFilteredLayerListRequest');
+if (reqBuilder) {
+    var request = reqBuilder(null, 'vectors', false);
+    sb.request('MainMapModule', request);
+}

```

Comment:
The example should be:

     Oskari.getSandbox().postRequestByName('ShowFilteredLayerListRequest', [null, 'vectors']);

In the original example sandbox.getRequestBuilder() is deprecated and Oskari.requestBuilder() should be used instead. Also since we are not a module sending the request it's better to use postRequestByName() instead of saying that it's MapModule sending the request. I guess there's a case to be made that the code is much shorter as well :)